src/Security/CheckGroups/CheckGroupDatabase.js

```diff
@@ -1,15 +1,12 @@
-/**
- * @module SecurityCheck
- */
-
 import { Check } from '../Check';
 import CheckGroup from '../CheckGroup';
 import Config from '../../Config';
 import Parse from 'parse/node';
 
 /**
  * The security checks group for Parse Server configuration.
- * Checks common Parse Server parameters such as access keys.
+ * Checks common Parse Server parameters such as access keys
+ * @memberof module:SecurityCheck
```

Comment:
I'm not sure whether this should actually be in the docs. These look like internal classes that the developer cannot use or configure. They are only used internally for a security feature.

Reply:
I wonder if it's possible to mark these headers with a "private" keyword and just omit the "private" items from the output? Because you annotation seems technically correct, so for internal docs it could make sense to have this properly assigned to a module.

Reply:
It's what I did, I keep the documentation adjustments and just remove the source file from doc output.

Reply:
Oh I didn't notice that, apologies